x/tier/keeper/keeper.go

```diff
@@ -146,7 +159,23 @@ func (k *Keeper) CompleteUnlocking(ctx context.Context) error {
 }
 
 // Lock locks the stake of a delegator to a validator.
-func (k *Keeper) Lock(ctx context.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress, amt math.Int) error {
+func (k *Keeper) Lock(ctx context.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress, amt math.Int) (err error) {
+	start := time.Now()
+
+	defer func() {
+		metrics.ModuleMeasureSinceWithCounter(
+			types.ModuleName,
+			metrics.Lock,
```

Comment:
suggestion: This is somewhat general to this PR, but highlighted here. Imo, the actual metrics instance (ie: `metrics.Lock`) should be defined within the respective modules. Otherwise, `metrics.Lock` isn't very descriptive without further context (A fuller name would be something like `metrics.TierDevLock`). If they were defined within the modules, for examplein the module `types` folder, it might make them a little clearer without need to be overly specific/descriptive for each name.

Not a blocker for this PR though.